src/ui/annotations.ts

```diff
@@ -1854,6 +1854,19 @@ export function UserLayerWithAnnotationsMixin<
 
                 const { relationships, properties } = annotationLayer.source;
                 const sourceReadonly = annotationLayer.source.readonly;
+                const globalCoordinateSpace =
+                  this.manager.root.coordinateSpace.value;
+                const defaultProperties = annotationTypeHandlers[
+                  annotation.type
+                ].defaultProperties(annotation, globalCoordinateSpace.scales);
```

Comment:
This is still not correct --- you need to transform the scales, taking into account the various affine transforms that apply, similar to what is done here:

https://github.com/google/neuroglancer/blob/5898236dbc127b46ea280fc75c9e37bcee03a49b/src/ui/annotations.ts#L1073